pkg/webui/console/containers/notifications-dashboard-panel/notifications-dashboard-panel.styl

```
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 .notification-panel
-  height: 26rem
+  height: 100%
```

```suggestion
  height: 100%
  min-height: 26rem
```
What's important is that all elements of the row will have the same height, which should be achieved through the grid. The `min-height` is more of a fallback just in case. In this PR for example the bookmark panel is missing, so the panel would be rendered too small.